packages/templates/clients/websocket/java/quarkus/README.md

```
@@ -7,10 +7,69 @@ You can test this template (for now modifications are coming):
+    --header 'Authorization: Bearer TAKE_XAPP_TOKEN_FROM_BOOKMARKS_DOC_IN_SLACK'
+    ```
+
+    Windows
+    ```powershell
+    curl.exe --location --request POST 'https://slack.com/api/apps.connections.open' `
+    --header 'Authorization: Bearer TAKE_XAPP_TOKEN_FROM_BOOKMARKS_DOC_IN_SLACK'
+    ```
+
+>**Note**:  Ensure that you do not expose the real token on GitHub or any other public platform because it will be disabled by Slack.
+
+    Example response with `ticket` and `app_id`:
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)